### PR TITLE
metrics: enable host OVS process metrics collection

### PR DESCRIPTION
**Pull request by dcbw**
Gather process metrics from the host OVS process using the default
pidfile locations.

### CODE REVIEW

File: go-controller/pkg/metrics/ovs.go [hunk 1]
```diff
@@ -1231,6 +1231,15 @@ func RegisterOvsMetrics() {
 		componentCoverageShowMetricsMap[ovsVswitchd] = ovsVswitchdCoverageShowMetricsMap
 		registerCoverageShowMetrics(ovsVswitchd, MetricOvsNamespace, MetricOvsSubsystemVswitchd)
 
+		prometheus.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{
+			PidFn:     prometheus.NewPidFileFn("/var/run/openvswitch/ovs-vswitchd.pid"),
```
Comment (dcbw) [inline go-controller/pkg/metrics/ovs.go:1235]: @girishmg are these pidfile locations the same for you? I think unless somebody builds a custom OVS they should be the same everywhere, but not sure we need/want to support custom OVS paths. We could do config options for that if needed though.

Comment (girishmg) [reply go-controller/pkg/metrics/ovs.go:1236]: @dcbw apologies for the delayed reply.. yes, our OVS uses `/var/run/openvswitch` for pidfiles. 

yes, if needed we can do config options.

we already do this in `ovnkube.sh`. So, in future we can expose the RUN directories for OVS and OVN.
```
# Determine the ovn rundir.
if [[ -f /usr/bin/ovn-appctl ]]; then
  # ovn-appctl is present. Use new ovn run dir path.
  OVN_RUNDIR=/var/run/ovn
  OVNCTL_PATH=/usr/share/ovn/scripts/ovn-ctl
  OVN_LOGDIR=/var/log/ovn
  OVN_ETCDIR=/etc/ovn
else
  # ovn-appctl is not present. Use openvswitch run dir path.
  OVN_RUNDIR=/var/run/openvswitch
  OVNCTL_PATH=/usr/share/openvswitch/scripts/ovn-ctl
  OVN_LOGDIR=/var/log/openvswitch
  OVN_ETCDIR=/etc/openvswitch
fi
```

Comment (dcbw) [reply go-controller/pkg/metrics/ovs.go:1236]: @girishmg OK to merge for now and add config options later if we find somebody using non-standard paths?

Comment (girishmg) [reply go-controller/pkg/metrics/ovs.go:1236]: ACK

**Pull request merged**